addons/mail/static/src/models/thread.js

```diff
@@ -2283,6 +2271,20 @@ registerModel({
             inverse: 'originThread',
             isCausal: true,
         }),
+        /**
+         * Registry of timers of partners currently typing in the thread,
+         * excluding current partner. This is useful in order to
+         * automatically unregister typing members when not receive any
+         * typing notification after a long time. Timers are internally
+         * indexed by partner records as key. The current partner is
+         * ignored in this registry of timers.
+         *
+         * @see registerOtherMemberTypingMember
+         * @see unregisterOtherMemberTypingMember
+         */
+        otherMembersLongTypingTimers: attr({
+            default: new Map(),
```

Comment:
should be compute to avoid every thread using the same map? I'm confused as to how it works like this actually

Reply:
I guess we never test the potential conflict between 2 channels typing at the same time

Reply:
Changes done